ClientApp/src/NavBar/NavBar.scss

```diff
@@ -80,6 +80,7 @@ li a:hover {
   right: 0;
   color: $light-text;
   z-index: 1001;
+  background-color: transparent !important;
```

Comment:
can we find a way to do this without using !important keyword? take a look at https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity on how to do this and maybe a warning on not using important :)